staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -426,6 +463,10 @@ func (s *APIAggregator) AddAPIService(apiService *v1.APIService) error {
 		if s.openAPIV3AggregationController != nil {
 			s.openAPIV3AggregationController.UpdateAPIService(proxyHandler, apiService)
 		}
+		// Forward calls to discovery manager to update discovery document
+		if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.AggregatedDiscoveryEndpoint) {
+			s.discoveryAggregationController.AddAPIService(apiService, proxyHandler)
```

Comment:
is this called on every update?  For example, is it called when status or labels are updated?